### DISCUSSION TITLE
Popover Tab tip

### DISCUSSION THREAD

**Discussion by sildeman** [carbon-design-system/carbon#11507] [Help] [answered]
Hello everybody. Quick question: how to implement a Popover with a "Tab tip", as presented in [this page](https://carbondesignsystem.com/components/popover/usage/#variants). 
I'm using the `@carbon/react` package.
Thanks!

**Comment by tay1orjones** [comment, marked as answer]
Hey @sildeman - Popover does not yet have a first-class config option for the tab tip, https://github.com/carbon-design-system/carbon/issues/11038

It's still possible to configure one though. You'd need `caret="false"` on the Popover and the trigger will need some custom styling to make it appear consistent with the tab tip specification.

As an example, for the [Popover story](https://react.carbondesignsystem.com/?path=/story/components-popover--playground&args=caret:false) we add some lightweight styling to the trigger:
https://github.com/carbon-design-system/carbon/blob/6a345aabaecef2caaff74d3ddb2e9a503537896b/packages/react/src/components/Popover/next/story.scss#L73-L84

**Reply by sildeman** [reply]
Thanks. I will try this solution.
